aiida/orm/implementation/groups.py

```diff
@@ -12,14 +12,15 @@
 import abc
 
 from aiida.common import exceptions
+from aiida.orm.implementation.entities import ExtrasBackendEntity
```

Comment:
Since this comes from the same package, it is nice to use relative import
```suggestion
from .entities import ExtrasBackendEntity
```
Makes it easier to read and easier to refactor should the package ever be moved